mkdocs_swan/notebook.html

```diff
@@ -1,4 +1,14 @@
- {% extends "full_width.html" %}
+<script type="text/javascript">
+    function openInSwan(){
+        if ( window.location !== window.parent.location ) {
+            window.parent.postMessage('{{config.gallery_url + page.meta.notebook_url | url}}','https://swan-qa001.cern.ch');
```

Comment:
Is there a way we can avoid hard-coding the URL second argument here? This would not work in other environments

Reply:
In this situation I don't see why we cannot use `*` (we're not passing secrets, so there's no security risks).
Otherwise, we should not import the value from a config, as SWAN can be server under any random URL.

Reply:
This would mean we would need to keep track of a list of origins. And this would need to be configured in the gallery website. i.e the same website will not work otherwise from different nodes.

Can we use `*` as the target origin here? In this case the message does not contain any sensitive information (just a public URL). So another website intercepting the message would not be a problem. (Although https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage mentions not to use `*`)

(On the other side in the JupyterLab extension we anyway verify that the message is coming from the correct origin `swan-gallery.cern.ch` which is where some malicious iframe can download files into the container.)

Reply:
(sorry for the duplicate comment, I had to refresh my page)